portal-api/src/v1/services/utilisation-report/email-service.test.ts

```diff
@@ -53,12 +57,23 @@ describe('emailService', () => {
 
     it('should throw an error if email recipients not provided', async () => {
       // Arrange
+      process.env.TFM_UI_URL = 'https://www.ukexportfinance.gov.uk';
       delete process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT;
 
       // Act + Assert
       await expect(sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam('My Bank', 'June 2026')).rejects.toThrow();
       expect(sendEmail).toHaveBeenCalledTimes(0);
     });
+
+    it('should throw an error if TFM_UI_URL is undefined', async () => {
```

Comment:
I think this should be tested as part of a unit test for `getTfmUiUrl` and then we should mock that function in this unit test

Reply:
Updated this function to use a spy thanks